distributive/distributive-marathon/distributive-marathon.json

```diff
@@ -0,0 +1,51 @@
+{
+    "Name": "marathon health check",
+    "Notes": "For use with Microservice Infrastructure",
+    "Checklist" : [
+        {
+            "Name": "marathon installation",
+            "ID": "installed",
+            "Parameters": ["marathon"]
+        },
+        {
+            "Name": "marathon wait script",
+            "ID": "file",
+            "Parameters": ["/usr/local/bin/marathon-wait-for-listen.sh"]
+        },
+        {
+            "Name": "marathon systemd unit file",
+            "ID": "file",
+            "Parameters": ["/usr/lib/systemd/system/marathon.service"]
+        },
+        {
+            "Name": "marathon collectd plugin",
```

Comment:
Same here. If collectd is configured incorrectly, it will impact the marathon service availability via consul. It needs to be a collectd-marathon check that is separate from the core service.